### DISCUSSION TITLE
Intel終於要吹起反攻的號角了嗎？

### DISCUSSION THREAD

**Discussion by ayakasanma** [1101-GS3073-C/Python-Community-GS3073-C#204] [General]
在高階處理器一直被壓制的Intel能不能透過徵才贏過AMD呢？

英特爾砸560億 全球大搶才

https://money.udn.com/money/story/5612/6026727?from=ednappsharing

**Comment by minniewei** [comment]
期待能夠超過amd 呀!!畢竟我的電腦cpu gpu都是intel的!!

**Comment by ghost** [comment]
聽說最近Intel出的12代CPU晶片大大超越了先前的幾代~期望Intel能夠超越

**Reply by Shiba010** [reply]
intell終於不要再擠牙膏了嗎？XD

**Comment by cobb110202008** [comment]
期待intel能開發出更好的處理器，畢竟現在我的電腦是amd處理器(但有蠻多問題的)。

**Reply by ayakasanma** [reply]
AMD很讚，但Windows的優化常常做的很爛（和intel有商業結盟）有時真的會氣到想換作業系統XD

**Comment by jack90325** [comment]
intel真的是市占率很高的公司，所以我希望他們能花更多經費在提升效能上

**Reply by ayakasanma** [reply]
他們公司業務太多，除了要應付台積電代工競爭還有AMD晶片的軍備競賽，執行長一定一個頭兩個大@@
